processor/spanmetricsprocessor/config.go

```diff
@@ -45,4 +46,13 @@ type Config struct {
 	// The dimensions will be fetched from the span's attributes. Examples of some conventionally used attributes:
 	// https://github.com/open-telemetry/opentelemetry-collector/blob/main/model/semconv/opentelemetry.go.
 	Dimensions []Dimension `mapstructure:"dimensions"`
+
+	aggregationTemporality string `mapstructure:"aggregation_temporality"`
```

Comment:
Wondering if there's a way to declare this directly as a `pdata.MetricAggregationTemporality` and not needing to convert it from string.